Makefile

```diff
@@ -3,17 +3,30 @@ VERSION  := 0.7.1
 SRC      := $(wildcard *.go)
 TARGET   := node_exporter
 
-OS   := $(subst Darwin,darwin,$(subst Linux,linux,$(shell uname)))
-ARCH := $(subst x86_64,amd64,$(shell uname -m))
-
-GOOS   ?= $(OS)
-GOARCH ?= $(ARCH)
-GOPKG  := go1.4.1.$(OS)-$(ARCH).tar.gz
-GOROOT := $(CURDIR)/.deps/go
-GOPATH := $(CURDIR)/.deps/gopath
-GOCC   := $(GOROOT)/bin/go
-GOLIB  := $(GOROOT)/pkg/$(GOOS)_$(GOARCH)
-GO     := GOROOT=$(GOROOT) GOPATH=$(GOPATH) $(GOCC)
+OS         := $(subst Darwin,darwin,$(subst Linux,linux,$(shell uname)))
+ARCH       := $(subst x86_64,amd64,$(shell uname -m))
+
+# The release engineers apparently need to key their binary artifacts to the
+# Mac OS X release family.
+MAC_OS_X_VERSION ?= 10.8
+
+GOOS       ?= $(OS)
+GOARCH     ?= $(ARCH)
+
+ifeq ($(GOOS),darwin)
+RELEASE_SUFFIX ?= -osx$(MAC_OS_X_VERSION)
+else
+RELEASE_SUFFIX ?=
+endif
+	
+GO_VERSION ?= 1.4.1
+GOURL      ?= http://golang.org/dl
```

Comment:
`https://golang.org/dl` also works with HTTPS. I'd prefer this "official" path rather than one which could change without notice...